services/catalog/src/simcore_service_catalog/services/access_rights.py

```diff
@@ -22,14 +22,16 @@
 
 _logger = logging.getLogger(__name__)
 
-OLD_SERVICES_DATE: datetime = datetime(2020, 8, 19)
+_LEGACY_SERVICES_DATE: datetime = datetime(
+    year=2020, month=8, day=19, tzinfo=timezone.utc
+)
 
 
 def _is_frontend_service(service: ServiceMetaDataPublished) -> bool:
     return "/frontend/" in service.key
 
 
-async def _is_old_service(app: FastAPI, service: ServiceMetaDataPublished) -> bool:
+async def _is_legacy_service(app: FastAPI, service: ServiceMetaDataPublished) -> bool:
```

Comment:
No, this is wrong.

This had nothing to do with legacy service, this had to do with service access rights. We decided years ago, that all services until that date would be available to everyone, and after that date would be only available to their respective authors.
Please revert that part, or check whether these are not any more in use (although since we still use sleepers 1.0.0 I doubt that this is the case.)